index.d.ts

```diff
@@ -1,9 +1,30 @@
 /// <reference types="node"/>
-import {JSONSchema} from 'json-schema-typed';
+import { JSONSchema } from 'json-schema-typed';
```

Comment:
Don't do unrelated style changes. Applies in many places. If you upgrade the XO dependency and run `npx xo --fix` these things will be fixed for you.